### ISSUE TITLE
pls updaye

### ISSUE THREAD

**Issue by Flonja** [ethaniccc/Esoteric#37]
my dog is very ill and needs protection only you cn help me pls
i will pay 23 robox pls

**Comment by Flonja** [comment]
i asked for help from discord but they talk about shooting my dog???????
thats very mean of them and i dont apriciate that 
i want to protect my dog from corona and am willing to payt 23 roblox for it!

**Comment by cqdetdev** [comment]
who let this slimy dutch girl speak

**Comment by MINTEDCLICKS** [comment]
> who let this slimy dutch girl speak

what if we made a cheat client like horion but its better

**Comment by Flonja** [comment]
mu dog is twitchin g rn what i do????

**Comment by ethaniccc** [comment]
commit die

**Comment by Flonja** [comment]
updatw i now threw my dog way and have a kitty now
what food doeds i t like???

**Comment by cqdetdev** [comment]
can someone tell this dutch fuck to shut the fuck up

**Comment by Flonja** [comment]
how did you knpw i am dutch???????????/
are you oudtside my window???

**Comment by cqdetdev** [comment]
![bg,f8f8f8-flat,750x,075,f-pad,750x1000,f8f8f8](https://github.com/ethaniccc/Esoteric/assets/101936396/fab8be04-94be-4374-b6bb-d75548de45f4)

**Comment by Flonja** [comment]
SIRRRRRRR thsi is nottt aoppriciated!!!!

**Comment by Flonja** [comment]
i think my kitty is deawsd rn i gave it glizy=s and it collaspsed
any other pet s i can get?????

**Comment by Flonja** [comment]
anyuway i need this project to preotect me next pet can u pls do that thx :))))

**Comment by cqdetdev** [comment]
big boy furry

**Comment by cqdetdev** [comment]
ethan shut up

**Comment by cqdetdev** [comment]
ger

**Comment by cqdetdev** [comment]
">:/"

**Comment by cqdetdev** [comment]
"furzide can you eat me"

**Comment by cqdetdev** [comment]
"furzide can you bark"

**Comment by cqdetdev** [comment]
"woof woof"

**Comment by cqdetdev** [comment]
<@377148912990945310>

**Comment by Flonja** [comment]
my caty is doing ok rn thanks for asking :) i swould still arppriciate a updat k thx bye

**Comment by Superomarking** [comment]
when is tis hapenning?????????????????? also how is the dog shooting process gign????/!

**Comment by Flonja** [comment]
dog??/// shooting?????? i thewr it wway mionths ago ?!?!?//??/

**Comment by Superomarking** [comment]
> dog??/// shooting?????? i thewr it wway mionths ago ?!?!?//??/

what shapepn>>?? with kitty ?!?! is tuid okaf????????????

**Comment by Flonja** [comment]
it ded ...esoteric coud have prefvented it... :(((((((((((

**Comment by Superomarking** [comment]
waow, im so sry for dsyou and eharign that, epls tekll if i can eat thes remains

**Comment by cqdetdev** [comment]
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣀⣀⣠⣤⣤⣀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣠⣤⣶⣾⣿⣿⣿⣿⣿⣿⣿⣿⣧⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣀⣤⣾⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⠀
⠀⠀⠀⠀⠀⠀⠀⢀⣴⣾⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⠀
⠀⠀⠀⠀⢀⣀⢾⣿⣿⣿⣿⣿⣿⣿⣿⣿⠿⠛⢋⣭⡍⣿⣿⣿⣿⣿⣿⠀
⠀⢀⣴⣶⣶⣝⢷⡝⢿⣿⣿⣿⠿⠛⠉⠀⠀⣰⣿⣿⢣⣿⣿⣿⣿⣿⣿⡇
⢀⣾⣿⣿⣿⣿⣧⠻⡌⠿⠋⠁⠀⠀⠀⠀⢰⣿⣿⡏⣸⣿⣿⣿⣿⣿⣿⣿
⣼⣿⣿⣿⣿⣿⣿⡇⠁⠀⠀⠀⠀⠀⠀⠀⠈⠻⢿⠇⢻⣿⣿⣿⣿⣿⣿⡟
⠙⢹⣿⣿⣿⠿⠋⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠈⠻⢿⣿⣿⡿⠟⠁
⠀⠀⠉⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
 ...
. .
.. ..
.
.
.

**Comment by Superomarking** [comment]
iam MINOR sirrr, loak at may plugni. codde ?? why it donts wonrk???????????
<img width="743" alt="image" src="https://github.com/ethaniccc/Esoteric/assets/71559522/13d86906-e3e6-477a-8174-df1a9c9476a8">

**Comment by GameParrot** [comment]
please help this kid called orangesweat keeps making new accounts on my server and cheating i cant do anything about it unless u update

**Comment by cqdetdev** [comment]
⠀ ⠀ ⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀ ⠀⠀⣀⣤⣤⣤⣤⣤⣤⣤⣤⣄⣀
⠀ ⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀ ⢀⣴⣾⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣶⣤⡀
⠀ ⠀⠀⠀⠀⠀⠀⠀⠀⠀ ⢠⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣦
⠀ ⠀⠀⠀⠀⠀⠀⠀⠀ ⢀⣾⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣷
⠀ ⠀⠀⠀⠀⠀ ⠀⠀⠀⣸⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⡿⠿⠿⠛⠛⠛⠛⠿⠿⣿⣿⣷⣄
⠀ ⠀⠀⠀ ⠀⠀⠀⠀⠀⣿⣿⣿⣿⣿⣿⣿⣿⣿⡆⠀⠀ ⠀⠀⠀⠀ ⠀ ⠀⠀⠀⠀⠈⠻⣿⣷
⠀⠀ ⢀⣠⣤⣴⣶⣶⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⡀⠀⠀ ⠀⠀⠀⠀⠀⠀⠀ ⠀ ⠀⠀⠀⢸⣿⡇
⠀⢀⣾⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣶⣤⣤⣤⣤⣤⣤⣤⣤⣴⣶⣿⣿⡿
⠀⢸⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⠃
⠀⢸⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿▄▄▀▀█⣿⣿⣿⣿⣿⠁
⠀⣸⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿▄█▒░░▄░█⣿⣿⣿⣿
⠀⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿▄▀▒▀▀▀▄▄▀⣿⣿⣿⣿
⠀⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿█▒░░░░░█⣿⣿⣿⣿⣿⣿
⠀⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿ █▒░░░░░█⣿⣿⣿⣿⣿⣿⡇
⠀⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿█▒▒░░░▒█⣿⣿⣿⣿⣿⣿⡿
⠀⢿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿▓▓▒▒▒▀▀▀█▄⣿⣿⣿⣿⣿⡇
⠀⢸⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿▓▒▒▒▒▒▒▒▒▒█⣿⣿⣿⣿⡇
⠀⠀⢿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿▓▒▒▒▒▒▒▓▒▒█⣿⣿⣿⣿
⠀⠀⠀⠙⠿⠿⠿⠿⣿⣿⣿⣿⣿⣿⣿⣿⣿▀▄▄▄▄█▄▄▀⣿⣿⣿⣿⣿
⠀⠀⠀⠀⠀⠀ ⠀⠀⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⡄⠀⣿⣿⣿⣿⣿⣿⣿⣿⣿⡇
⠀⠀⠀⠀⠀⠀⠀ ⠀⢸⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣷⢐⣿⣿⣿⣿⣿⣿⣿⣿⣿⠃
⠀⠀⠀⠀⠀⠀⠀ ⠀⠀⢻⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣾⣿⣿⣿⣿⣿⣿⣿⣿⣿
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢻⣿⣿⣿⡿⣟⣯⣿⠟⡉⠉⣿⣿⣿⣿⣿⣿⣿⣿⣿
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠻⢿⣽⣿⣿⣿⠿⠿⠟⠒⠉⠉⠉⠉⠉⠉⠉⠙⠋
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠈⠿⠋⠉⢀⣠⣤⣤⡔⣄
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣴⠾⠛⠋⠉⠀⢀⣀⠐⣤⣶⣶⡤⢤⣤
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣤⣰⣶⣾⣿⣿⣿⣆⠀⣀⣀⡀⣀⡀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠉⠉⠀⢀⢀⣀⠀⣀⣈⡿⠿⠿⠽⠃
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠛⠛⠿⠿⠿⠿⠾⠟⢁⣀⡴⣦⠆
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢦⣤⣀⣀⠀⠀⠀⠀⢘⣿⣍⡷⠆
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢶⣄⠈⠉⠛⠛⠿⠓⠀⠉⠋⠉⣀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣧⡀⠙⠻⢶⣶⡤⠀⠀⠛⠶⠾⠼⠋
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣆⠈⠻⣶⣤⡀⠀⠀⢸⠿⣶⣦⣤⣠⣾
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢠⠙⢷⣤⣀⠈⠁⠀⠀⢠⣤⣀⠈⠉⠈
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⡌⢧⣀⠉⠛⠃⠀⠀⠀⠀⠉⠛⠿⠿⠻⠃
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠰⢳⣄⠙⠛⢋⠁⠀⠀⠀⠘⠿⣴⣤⣄⣤⡄
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠘⣄⡙⠛⠋⠀⠀⠀⠀⠀⠰⣤⣀⠉⠉⠉⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⢠⡈⠉⠉⠀⠀⠀⠀⠀⠀⢀⡈⠙⠛⠛⠛⠁
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠈⢦⡉⠛⡁⠀⠀⠀⠀⠀⠀⠈⠻⠷⣶⣦⡆
⠀⠀⠀⠀⠀⠀⠀⠀⢠⡈⢷⣌⠙⠛⠁⠀⠀⠀⠀⠀⠀⠰⣦⣄⣀⣀⡀
⠀⠀⠀⠀⠀⠀⠀⠀⠈⢷⣄⡉⠛⠛⠀⠀⠀⠀⠀⠀⠀⢀⠈⠙⠛⠛⠀
⠀⠀⠀⠀⠀⠀⠀⠀⢦⣀⠉⠛⠷⠖⠀⠀⠀⠀⠀⠀⠀⠘⠿⣶⣦⡄
⠀⠀⠀⠀⠀⠀⠀⣠⣀⠙⠳⠶⠶⠀⠀⠀⠀⠀⠀⠀⠀⢠⣀⣀⣀
⠀⠀⠀⠀⠀⠀⠀⠙⠻⢿⣶⣤⣤⠀⠀⠀⠀⠀⠀⠀⢠⠛⠛⠻⠿
⠀⠀⠀⠀⠀⠀⠀⠰⣦⣄⠈⠉⠋⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠈
⠀⠀⠀⠀⠀⠀⠀⠀⢹⣿⣿⣶⡆⠀⠀⠀⠀⠀⠀⠀⠺⠿⠿⠿⠁
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠙⠻⠟⠁⠀⠀⠀⠀⠀⠀⢀⣤⣤⣤⣤⡄
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠈⣀⣀⣀⣀⠁
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠛⠛⠻⠿⠿⠧
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢠⣞⣻⣿⣿⣔⣿⠂
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠈⠛⠋⠉⠉⠁

**Comment by cqdetdev** [comment]
⣿⣿⡻⠿⣳⠸⢿⡇⢇⣿⡧⢹⠿⣿⣿⣿⣿⣾⣿⡇⣿⣿⣿⣿⡿⡐⣯⠁ ⠄⠄
⠟⣛⣽⡳⠼⠄⠈⣷⡾⣥⣱⠃⠣⣿⣿⣿⣯⣭⠽⡇⣿⣿⣿⣿⣟⢢⠏⠄ ⠄
⢠⡿⠶⣮⣝⣿⠄⠄⠈⡥⢭⣥⠅⢌⣽⣿⣻⢶⣭⡿⠿⠜⢿⣿⣿⡿⠁⠄⠄
⠄⣼⣧⠤⢌⣭⡇⠄⠄⠄⠭⠭⠭⠯⠴⣚⣉⣛⡢⠭⠵⢶⣾⣦⡍⠁⠄⠄⠄⠄
⠄⣿⣷⣯⣭⡷⠄⠄⢀⣀⠩⠍⢉⣛⣛⠫⢏⣈⣭⣥⣶⣶⣦⣭⣛⠄⠄⠄⠄⠄
⢀⣿⣿⣿⡿⠃⢀⣴⣿⣿⣿⣎⢩⠌⣡⣶⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣆⠄⠄⠄
⢸⡿⢟⣽⠎⣰⣿⣿⣿⣿⣿⣿⢀⣾⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣦⠄⠄
⣰⠯⣾⢅⣼⣿⣿⣿⣿⣿⣿⡇⣾⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⡄⠄
⢰⣄⡉⣼⣿⣿⣿⣿⣿⣿⣿⢸⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣧⠄
⢯⣌⢹⣿⣿⣿⣿⣿⣿⣿⣿⢸⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⠄
⢸⣇⣽⣿⣿⣿⣿⣿⣿⣿⣿⠸⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⠄
⢸⣟⣧⡻⣿⣿⣿⣿⣿⣿⣿⣧⡻⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⠄
⠈⢹⡧⣿⣸⠿⢿⣿⣿⣿⣿⡿⠗⣈⠻⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⡇⠄
⠄⠘⢷⡳⣾⣷⣶⣶⣶⣶⣶⣾⣿⣿⢀⣶⣶⣶⣾⣿⣿⣿⣿⣿⣿⣿⣿⣿⠇⠄
⠄⠄⠈⣵⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⢸⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⠄⠄
⠄⠄⠄⠸⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⠘⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⠇⠄⠄

**Comment by Superomarking** [comment]
SiR, WHERE ARE thE BoobsIES????????????????????

**Comment by GameParrot** [comment]
plz help ur bf thinks nslith cheats and i want to find out if thats true
i will pay 420 bobux

**Comment by Superomarking** [comment]
ill do 420 + 69 robbulxx

**Comment by GameParrot** [comment]
> ill do 420 + 69 robbulxx


i will do 69420 bobux, 420 robux, and 69 vbucks

**Comment by Superomarking** [comment]
I WIll bUY The forTNITE BATTLE BASS +++++++ 20 VBUCKS ++ MINEXCAFT PRO GAMER SKIN +_ 42069 robux

beat that !!!!!!!!!!!!!!!!!

**Comment by GameParrot** [comment]
Regarding Future of Third Party Software on MCPE
This message concerns all multiplayer players. In short, Mojang has decided to strip out symbols/PDB from future builds of MCPE. The goal was the symbols was to provide third-party Bedrock developers similar tooling to Java mappings to be able to create software on Bedrock (Pocketmine uses this specifically for block palettes and upgrading). Due to Mojang’s genius decision to remove them, this is going essentially kill all third-party development (except to those with the PDB under an NDA).

I’m urging you all to reply to this forum:

https://feedback.minecraft.net/hc/en-us/community/posts/23093086772109-Request-for-Complete-PDB-for-Minecraft-Bedrock-Dedicated-Server

Although change is unlikely, we hope to create somewhat of a dent. 

In short, new updates to MCBE will make it tedious to update and kill basically all third party software

@everyone

**Comment by didntpot** [comment]
Regarding Future of Third Party Software on MCPE
This message concerns all multiplayer players. In short, Mojang has decided to strip out symbols/PDB from future builds of MCPE. The goal was the symbols was to provide third-party Bedrock developers similar tooling to Java mappings to be able to create software on Bedrock (Pocketmine uses this specifically for block palettes and upgrading). Due to Mojang’s genius decision to remove them, this is going essentially kill all third-party development (except to those with the PDB under an NDA).

I’m urging you all to reply to this forum:

https://feedback.minecraft.net/hc/en-us/community/posts/23093086772109-Request-for-Complete-PDB-for-Minecraft-Bedrock-Dedicated-Server

Although change is unlikely, we hope to create somewhat of a dent.

In short, new updates to MCBE will make it tedious to update and kill basically all third party software

@everyone

**Comment by cjmustard** [comment]
owo

**Comment by Superomarking** [comment]
XDekai, a reprehensible and depraved individual, has committed an endless list of heinous crimes that would make the most seasoned law enforcement officer's stomach turn. His depravity knows no bounds as he has repeatedly violated the basic human rights of innocent children, locking over 100 of them in his vile basement and subjecting them to unspeakable acts of brutal molestation. This monstrous behavior is just the tip of the iceberg, as XDekai is also responsible for the atrocious acts of animal cruelty, slavery, and genocide. His insatiable thirst for depravity extends even beyond the living, as he is a necrophiliac. The very thought of this man's existence sends shivers down the spine of any decent human being, and we can only hope that justice will soon be served and XDekai will be held accountable for his unimaginable crimes.

**Comment by Superomarking** [comment]
when fix anticheat?? latest version no work, i get internet server error + ethanicc isn't sendming me oomhp sourc to make into pocjketmine-mp plugin

**Comment by Flonja** [comment]
1.png 2.png 3.png 4.png @everyone @here

**Comment by NopeNotDark** [comment]
5.png 6.png 7.png wait SIX SEVEN @here

**Comment by cjmustard** [comment]
123 for bf

**Comment by NopeNotDark** [comment]
please stop here 6.png 7.png

**Comment by Flonja** [comment]
> 123 for bf

123
